app-android/src/main/java/com/ustadmobile/port/android/view/ReportListFragment.kt

```diff
@@ -49,7 +58,16 @@ class ReportListFragment() : UstadListViewFragment<Report, Report>(),
             holder.itemView.setSelectedIfInList(item, selectedItems, DIFF_CALLBACK)
             (holder.itemBinding.listReportChart.getTag(R.id.tag_graphlookup_key) as? Job)?.cancel()
             val graphJob = GlobalScope.async(Dispatchers.Main) {
-                val chartData = presenter?.getGraphData(item)
+                val series = if(!item.reportSeries.isNullOrEmpty()){
```

Comment:
I think when we are parsing a JSON we should probably use try/catch. Just in case one chart data gets corrupt somehow (e.g. if we change options etc) we don't want the app to crash or to break other reports.